lib/generators/boring/honeybadger/install/install_generator.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Boring
+  module Honeybadger
+    class InstallGenerator < Rails::Generators::Base
+      source_root File.expand_path("templates", __dir__)
+      desc 'Adds honeybadger to the app'
+
+      class_option :api_key, type: :string, aliases: '-k', default: 'key', desc: 'Tell us the honeybadger api key for the application'
```

Comment:
I think it would be better to ask user if they want to use env variables or rails credentials.
And use this variable instead of directly passing api key

and at the end show message for user to add respective variables
for reference:  https://github.com/TruemarkDev/boring_generators/pull/5